src/ports/biconomy/component.ts

```diff
@@ -0,0 +1,129 @@
+import { BigNumber } from 'ethers'
+import { parseUnits } from 'ethers/lib/utils'
+import { ErrorCode } from 'decentraland-transactions'
+import { ChainId } from '@dcl/schemas'
+import { AppComponents } from '../../types'
+import { TransactionData } from '../../types/transactions/transactions'
+import {
+  BiconomyMetaTransactionComponent,
+  GasPriceResponse,
+  MetaTransactionRequest,
+  MetaTransactionResponse,
+  MetaTransactionStatus,
+  toErrorCode,
+} from './types'
+import { InvalidTransactionError } from '../../types/transactions/errors'
+
+export function createBiconomyComponent(
+  components: Pick<AppComponents, 'config' | 'fetcher' | 'logs' | 'metrics'>
+): BiconomyMetaTransactionComponent {
+  const { config, fetcher, metrics, logs } = components
+  const logger = logs.getLogger('biconomy')
+
+  async function sendMetaTransaction(
+    transactionData: TransactionData
+  ): Promise<string> {
+    const biconomyAPIId = await config.requireString('BICONOMY_API_ID')
```

Comment:
nick-pick, can we move this `requireString` outside this fn so they are not retrieve on each `sendMetaTransaction` call?

Reply:
Changing this breaks all the tests because of the test runner. I'll send this change in another PR, as it is quite big.